docs/platform/howto/use-billing-groups.rst

```diff
@@ -1,63 +1,29 @@
-Manage billing groups in the Aiven Console
-==========================================
-
-Billing groups enable you to set a common billing profile for multiple
-projects and generate a consolidated invoice. To know more about the key benefits read the :doc:`related documentation <../concepts/billing-groups>`.
-
-Billing groups and Aiven accounts
----------------------------------
-
-An Aiven **Account** is the entry point for creating and managing billing groups. The following are options available for billing groups via the `Aiven Console <https://console.aiven.io/>`_:
-
-- Within an Account you can create one or more billing groups, depending on the need
-- You can group projects under an Account into one or more billing groups
-- You can set a **primary billing group** for an Account. All future projects associated with this Account are then added to the primary billing group by default.
-- You can move projects under an Account from one billing group to another.
-
-Aiven credits are available for all projects associated with the Account. You can choose to apply the credits to a specific project within the Account.
-
 Manage billing groups
----------------------
-
-To view and manage billing groups in the `Aiven Console <https://console.aiven.io/>`_:
-
-#. Click the current project and select **See all projects & accounts**.
+=====================
 
-#. Click an account that you want to manage and then click the **Billing** tab.
+View and manage billing groups in the Aiven Console
+---------------------------------------------------
 
-   .. Tip::
-    This window allows you to create new billing groups and assign projects to your billing groups. For more information on creating billing groups, `see the related article <https://help.aiven.io/en/articles/4634847-getting-started-with-billing-groups>`__.
+#. In the account that you want to manage, click **Billing**.
 
 #. Click one of the listed billing groups to browse the details for that billing group.
 
-Navigate the billing group page
-'''''''''''''''''''''''''''''''
-
-The billing group page contains multiple sections:
-
-* the **accumulated costs** of all projects that are assigned to the billing group in the top-right corner.
-* the *action menu* , available in the top-right corner, includes options to **rename or delete the billing group**. 
+Billing group page overview
```

Comment:
I'd replace this section (which most likely could be added as a concept judging by the title) with a few how-to sections covering, e.g.:
- Rename a billing group
- Delete a billing group
- And so on for all the relevant tabs.
These can be really short and simplified - what I mean is that the user expect to get some instructions here.
Ofc I'd also sneak in some general info on the page using notes, for example.